src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -194,6 +196,115 @@ tags:
       }
       ```
 
+      ## Consistent reads
+
+      As described in [Browsing historical data](#section/Browsing-historical-data) in all Network Gateway requests it is possible to specify the exact ledger state by providing the `at_ledger_state` parameter.
```

Comment:
This is not 100% true. Could we please clarify that this is applicable to all `/state/*` and `/stream/*` endpoints and in case of other areas it should be read from endpoint docs?